<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonDialog</class>
 <widget class="QDialog" name="PersonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Person</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layoutMain" stretch="1,2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelId">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineId">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelGroup">
         <property name="text">
          <string>Group</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboGroup">
         <item>
          <property name="text">
           <string>- None -</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tablePictures">
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>24</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>24</number>
       </attribute>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Path</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>O</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>M</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Action</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddPicture">
       <property name="text">
        <string>Add picture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineId</tabstop>
  <tabstop>lineName</tabstop>
  <tabstop>tablePictures</tabstop>
  <tabstop>buttonAddPicture</tabstop>
  <tabstop>buttonOk</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonAddPicture</sender>
   <signal>clicked()</signal>
   <receiver>PersonDialog</receiver>
   <slot>addPicture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>PersonDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addPicture()</slot>
 </slots>
</ui>
